rust-src/ecvrf/src/errors.rs

```diff
@@ -52,13 +36,6 @@ impl ::failure::Fail for InternalError {}
 ///   constructed from 255-bit integers.)
 ///
 /// * Failure of a proof to satisfy the verification equation.
-#[derive(Clone, Copy, Eq, PartialEq, Hash, Debug)]
+#[derive(Clone, Copy, Eq, PartialEq, Hash, Debug, Error)]
+#[error("{0}")]
```

Comment:
If the intent is to hide the ProofError then we should use `error(transparent)`.

But I think the idea would be not to hide the fact that this is a ProofError so the display should indicate that.